default.json

```diff
@@ -9,7 +9,7 @@
   "automergeStrategy": "merge-commit",
   "prConcurrentLimit": 0,
   "constraints": {
-    "go": "1.19"
+    "go": "1.20"
```

Comment:
Not sure if this one is needed.

Reply:
Unless I misread the [docs](https://docs.renovatebot.com/configuration-options/#constraints), this would not apply to Go:

> Constraints are also used to manually restrict which datasource versions are possible to upgrade to based on their language support. For now this datasource constraint feature only supports python, other compatibility restrictions will be added in the future.